docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -248,6 +248,79 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[filterNow]
 Currently, using a _precision_ greater than 3 doesn't make any difference to the output of the
 function as the maximum number of second fractional digits returned is 3 (milliseconds).
 
+[[sql-functions-datetime-add]]
+==== `DATE_ADD/DATE_ADD/TIMESTAMPADD/TIMESTAMP_ADD`
+
+.Synopsis:
+[source, sql]
+--------------------------------------------------
+DATE_PART(
+    string_exp, <1>
+    integer_exp, <2>
+    datetime_exp) <3>
+--------------------------------------------------
+
+*Input*:
+
+<1> string expression denoting the unit to add to the date/datetime
+<2> integer expression denoting how many times the above unit should be added/subtracted (if it's negative)
+to/from the date/datetime
+<3> date/datetime expression
+
+*Output*: datetime
+
+.Description:
+
+Add/Subtract the given number of units to/from a date/datetime. If any of the three arguments is `null`
```

Comment:
Again, it's `DATEADD` - I get it you can effectively subtract by using negative values by using `Add/Subtract` implies one can do subtraction of a negative value as well which is not the case.
In other words there's only addition of both positive and negative values but never a subtraction.